repos/runtime.common.props

```diff
@@ -0,0 +1,145 @@
+      <LibrariesBuildArguments Condition="'$(OfflineBuild)' == 'true'">$(LibrariesBuildArguments) /p:DotNetSourceBuildIntermediatePath=$(GeneratedSourcePathOffline)</LibrariesBuildArguments>
+      <LibrariesBuildArguments Condition="'$(OfflineBuild)' == 'true'">$(LibrariesBuildArguments) /p:MicrosoftNETCoreDotNetHostPackageVersion=$(DOTNET_HOST_BOOTSTRAP_VERSION)</LibrariesBuildArguments>
+      <LibrariesBuildArguments Condition="'$(OfflineBuild)' == 'true'">$(LibrariesBuildArguments) /p:MicrosoftNETCoreDotNetHostPolicyPackageVersion=$(DOTNET_HOST_BOOTSTRAP_VERSION)</LibrariesBuildArguments>
+
+      <LibrariesBuildArguments>$(LibrariesBuildArguments) $(FlagParameterPrefix)verbosity $(LogVerbosity)</LibrariesBuildArguments>
+
+      <!--
+      Disable node reuse (in the new/Arcade way) to avoid errors like this:
+
+      error MSB4062: The "Microsoft.CodeAnalysis.BuildTasks.Csc" task could not be loaded from the
+      assembly .../build/../tools/Microsoft.Build.Tasks.CodeAnalysis.dll. Assembly with same name
+      is already loaded
+      -->
+      <LibrariesBuildArguments>$(LibrariesBuildArguments) $(FlagParameterPrefix)nodereuse $(ArcadeFalseBoolBuildArg) $(FlagParameterPrefix)ci</LibrariesBuildArguments>
+
+      <!-- installer -->
+
+      <InstallerBuildArguments>$(FlagParameterPrefix)restore $(FlagParameterPrefix)build</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) $(FlagParameterPrefix)configuration $(Configuration)</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) $(FlagParameterPrefix)ci</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) /p:PortableBuild=$(OverridePortableBuild)</InstallerBuildArguments>
+      <InstallerBuildArguments Condition="$(Platform.Contains('arm')) AND '$(BuildArchitecture)' != 'arm64'">$(InstallerBuildArguments) /p:TargetArchitecture=$(Platform) /p:DisableCrossgen=true /p:CrossBuild=true</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) /p:BuildDebPackage=false</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) /p:BuildAllPackages=true</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) /p:RestoreAllBuildRids=false</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) /p:OutputRid=$(OverrideTargetRid)</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) /p:DotNetOutputBlobFeedDir=$(SourceBuiltBlobFeedDir)</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) /p:BuiltSdkPackageOverride=""</InstallerBuildArguments> <!-- Clear PackageOverride for Installer - it only applies to coreclr -->
+      <InstallerBuildArguments>$(InstallerBuildArguments) /m:1</InstallerBuildArguments>
```

Comment:
This could be masking some build parallelism issues and slow down the build, why is it here?

Reply:
This was added in Oct in 3.1 in a commit with the comment "remove parallel builds".  See https://github.com/dotnet/source-build/pull/1237/commits/31c88979509a9623e48cc0cee8a6637147bc2944
If we don't think it's needed any more, I'll remove it.
@crummel any idea what the issue was?

Reply:
Sorry, didn't see it was already there. Leaving it for later seems fine.

Reply:
I think we added this while running into the Roslyn compiler server hash mismatch issues, it probably shouldn't be needed since we figured out the actual fix for that.  Have you ever tried a build with the new fix and not `/m:1` @dagood?

Reply:
If we think it's fixed, I'll remove it.

Reply:
Not as far as I know, no. Filed https://github.com/dotnet/source-build/issues/1551 for followup.